logline.go

```diff
@@ -205,3 +205,38 @@ func (l *LogLine) Meta(key string) (interface{}, bool) {
 func (l *LogLine) Cache() *Cache {
 	return &l.cache
 }
+
+// Copy creates and returns a copy of the current log line.
+func (l *LogLine) Copy() Record {
+	// Copy metadata
+	md := make(Metadata)
+	for k, v := range l.meta {
+		md[k] = v
+	}
+
+	cpy := &LogLine{
+		cache:          l.cache,
+		meta:           md,
+		FieldSeparator: l.FieldSeparator,
+	}
+
+	if l.wcnt != 0 {
+		// If the log line has been modified, benchmarks have proven
+		// that it's more efficient to serialize and reparse to perform
+		// a copy (both in terms of time and allocation.
+		cpylen := len(l.data) + len(l.data)/2
+		text := l.ToText(make([]byte, 0, cpylen))
+		cpy.Parse(text, md)
+
+		return cpy
+	}
+
+	// If the log line hasn't been modified it's more efficient to recreate it
+	// from scratch and copying data (log line internal buffer).
+	if l.data != nil {
+		cpy.wcnt = l.wcnt
```

Comment:
I think this is useless, `l.wcnt` is zero as well as `cpy.wcnt`

Reply:
ok, I see you removed it 👍

Reply:
Yeah sorry😂